server/xpub/entities/manuscript/resolvers.test.js

```diff
@@ -281,9 +281,13 @@ describe('Submission', () => {
         ),
         encoding: 'utf8',
         mimetype: 'application/pdf',
+        size: 73947,
       }
-
-      const manuscript = await Mutation.uploadManuscript({}, { id, file })
+      const manuscript = await Mutation.uploadManuscript(
+        {},
+        { id, file },
+        file.size,
```

Comment:
Is this right? Shouldn't file size be passed in the second parameter?

Reply:
I don't remember what happened here, pretty sure I tested a while ago. This will fail anyway now because you need to subscribe to uploadProgress before you upload a file. I would skip for now and fix the test later since merging this seems like a higher priority.

Reply:
https://github.com/elifesciences/elife-xpub/issues/493